model/feature_flags.go

```diff
@@ -20,13 +20,17 @@ type FeatureFlags struct {
 	CollapsedThreads bool
 	// Feature flags to control plugin versions
 	PluginIncidentManagement string `plugin_id:"com.mattermost.plugin-incident-management"`
+
+	// AppsEnabled toggle the Apps framework functionalities both in server and client side
+	AppsEnabled bool
 }
 
 func (f *FeatureFlags) SetDefaults() {
 	f.TestFeature = "off"
 	f.TestBoolFeature = false
 	f.CloudDelinquentEmailJobsEnabled = false
 	f.CollapsedThreads = false
+	f.AppsEnabled = true
```

Comment:
Typically a feature starts in the "disable" (default) state and after one gets enough confidence wrt to its functioning then it gets permanently set to the "enable" state or the feature flag is removed - though maybe it doesn't make much of a difference now since the number of cloud deployments is relatively small. This is different from a killswitch for which the new behavior is by default "on" and when the killswitch is turned "on" the old behavior is "on" . Do we know what is the mechanism for turning the feature flag "on" on a subset of the cloud instances?